config/rbac/flagd_kubernetes_sync_clusterrole.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+# Prepends to open-feature-operator-flagd-kubernetes-sync
+  name: flagd-kubernetes-sync
+rules:
+- apiGroups: ["core.openfeature.dev"] # "" indicates the core API group
```

Comment:
`# "" indicates the core API group`

I don't understand this comment.

Reply:
It's something I should remove `- apiGroups: ""` would indicate core API group, will update